src/lib/structures/InhibitorStore.js

```diff
@@ -26,7 +26,19 @@ class InhibitorStore extends Store {
 	 */
 	async run(message, command, selective = false) {
 		const mps = [];
-		for (const inhibitor of this.values()) if (inhibitor.enabled && (!selective || !inhibitor.spamProtection)) mps.push(inhibitor.run(message, command).catch(err => err));
+
+		for (const inhibitor of this.values()) {
```

Comment:
For consistency with other structures, each call should be done in another method, to avoid re-creating a local function on every call, making inhibitors more expensive than what they currently are.

For example, MonitorStore:

https://github.com/dirigeants/klasa/blob/3bb3000dafac8a35470b6d6a6c8b4847cca66f6a/src/lib/structures/MonitorStore.js#L19-L41

Reply:
shoot, you are right...

Reply:
Additionally, I'd like to discriminate throws with string, and throws with the Error constructor, so when there's an inhibitor throwing a legit error (e.g. breaking changes affecting one inhibitor), it doesn't get sent to the channel without logging any of the stack trace. Though this would make this PR semver-major...